Shoko.Server/Services/Ogg/OggFile.cs

```diff
@@ -0,0 +1,463 @@
+using System;
```

Comment:
Would it be worth reformatting this to match with the `.editorconfig` setup?

Reply:
Done. Used Format Document from Visual Studio.